packages/gawk.rb

```diff
@@ -37,21 +25,26 @@ def self.patch
   end
 
   def self.build
-    system 'autoreconf -fvi'
-    system 'filefix'
+    system './bootstrap.sh'
+    # For some reason, ./bootstrap.sh doesn't call autoconf at all
+    system 'autoreconf -fiv'
     system "./configure #{CREW_OPTIONS}"
     system 'make'
   end
 
   def self.check
     # Still has failing issues on i686.
-    system 'make check || true'
+    case ARCH
+    when 'i686'
+      system 'make check || true'
+    else
+      system 'make', 'check'
+    end
   end
 
   def self.install
-    # Conflict with /usr/local/bin/awk from mawk package.
-    ENV['CREW_CONFLICTS_ONLY_ADVISORY'] = '1'
-    reload_constants
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    # Remove conflict with #{CREW_PREFIX}/bin/awk from mawk package
+    FileUtils.rm "#{CREW_DEST_PREFIX}/bin/awk"
```

Comment:
Is gawk better than mawk?  Should we remove the conflict from malk instead?

Reply:
gawk has more features but its slower. we use mawk by default in buildessential for that reason, so here mawk is the default as well.